apps/namadillo/src/hooks/useTransaction.tsx

```diff
@@ -33,12 +32,11 @@ type PartialNotification = Pick<ToastNotification, "title" | "description">;
 export type UseTransactionPropsEvents<T> = {
   onSigned?: (tx: TransactionPair<T>) => void;
   onError?: (err: unknown) => void;
-  onSuccess?: (tx: TransactionPair<T>) => void;
```

Comment:
Removed to improve clarity. Success will only happen when the transaction is actually applied, not after being broadcasted to the chain.

Reply:
great idea! I was thinking about it on the custom events as well, renaming them to `Broadcasted` and `Applied`